garage/tf/samplers/off_policy_vectorized_sampler.py

```diff
@@ -151,12 +192,13 @@ def process_samples(self, itr, paths):
         """
         success_history = []
         for path in paths:
```

Comment:
I think you forgot to remove this block of code. `success_history` is assigned again below.

Reply:
Nice catch!